images/scripts/fedora.setup

```diff
@@ -155,11 +155,14 @@ echo "config_opts['use_bootstrap'] = False" >>/etc/mock/site-defaults.cfg
 su builder -c "/usr/bin/mock --no-bootstrap-chroot --verbose -i $(/var/lib/testvm/build-deps.sh "$opsys $version")"
 su builder -c "/usr/bin/mock --install --verbose rpmlint"
 
-# we need to ensure that mock's selinux-policy is the same as the host
+# we need to ensure that mock's selinux-policy is older than the host:
+# disabling the updates repository will surely get us a very old version
 mkdir /tmp/selinux-policy-rpms
-dnf download --downloaddir /tmp/selinux-policy-rpms selinux-policy{,-devel,-targeted}
-dnf install -y /tmp/selinux-policy-rpms/*.rpm
+selinux_pkgs="selinux-policy selinux-policy-devel selinux-policy-targeted"
+dnf download --downloaddir /tmp/selinux-policy-rpms --disablerepo=updates $selinux_pkgs
```

Comment:
I think this will still fail on the `fedora-testing` image -- for that you'd also need to disable the `updates-testing` repo. Perhaps it'd be cleaner to do `--repo=fedora`? AFAIUI, that should use *only* that repo.

Reply:
Thanks for the tip.